### ISSUE TITLE
Debug messages left over in Point3D ?

### ISSUE THREAD

**Issue by tom-berend** [jsxgraph/jsxgraph#673]
I am finding some strange lettering on the screen when I put up LOTS of Point3D elements.  It does not happen with just a few points.  Is this perhaps some leftover debugging?

[live demo here](https://communityreading.org/jsxgraph/3d.html)

![2024-06-20_15-40](https://github.com/jsxgraph/jsxgraph/assets/11073753/b6df9c75-ce2a-4ebe-b707-846fe3f2da48)

**Comment by tom-berend** [comment]
If it's any help, it is coming from text.js
![2024-06-20_17-40](https://github.com/jsxgraph/jsxgraph/assets/11073753/50834aa6-a634-44b0-b1f0-f1c16dfd1d66)

**Comment by alfredwassermann** [comment]
Thank you very much for pointing this out. Actually it is a bug but its nature is a little bit subtle.

TL;DR: use the attribute `{withLabel: false}`

Here is the explanation:

First, the point labels are shown intentionally. However, the names are incorrect. By default, points and 3D points are labeled automatically with names from [A-Z]. If you want a different name for such a point, you have to supply the attribute `name`. If no name should be displayed, one could set `name:''` or use the more efficient `withLabel:false`. 

Second, the current JSXGraph contains a regression: when choosing the name, 3D points are not recognized and the generic names `s_a`, ... instead of `A`, ... are chosen. This is fixed now in branch `develop`.

Third, one may wonder why the labeling stops at `Z` or `s_z`? This is a design decision of JSXGraph. It was decided some years ago that if a construction has label names beyond `Z`, the labels will not contribute any longer to the understanding of the situation. So, we stop at `Z` or `s_Z`.

Best wishes,
Alfred
